airbyte-webapp/src/components/ui/Breadcrumbs/Breadcrumbs.module.scss

```diff
@@ -0,0 +1,21 @@
+@use "scss/colors";
+
+.container {
+  font-weight: normal;
+  cursor: default;
+}
+
+.item {
+  display: inline-block;
+  cursor: pointer;
+  color: colors.$blue-400;
+  text-decoration: none;
+
+  &--unlinked {
```

Comment:
scss modules do a really great job of isolating classes. if you made this `.unlinked` and provided `styles.item, styles.unlinked` to it it may play more nicely with the react code.

Reply:
Yeah fair, I just have a knee-jerk reaction to unstructured CSS class names 😁 Even with scoped classes I think it's helpful to see which classes are related/modifier classes when I'm scanning a `*.module.scss` file. But there is really no reason for me to introduce a new naming convention here, so I'll stick to simple class names. You're right that they play more nicely with react & `classnames`!

Reply:
Updated ✔️